README.md

```diff
@@ -35,29 +35,98 @@ worker.commands
   })
 ```
 
-## Custom message
+
+
+
+## Creating a custom message
 
 When using `permissionsMiddleware()` you can pass a custom message object with a function, that takes a function which takes the command context:
 ```js
 {
-  user: (ctx) => ...,//
-  my: (ctx) => ... //
+  user: (perms, ctx) => ...,//
+  my: (perms, ctx) => ... //
 }
 ```
 
 By default it is 
 ```js
 {
-  user = (ctx) => `You're the following permissions: ${ctx.command.userPerms.join(', ')}`,
-  my = (ctx) => `I'm missing the following permissions: ${ctx.command.myPerms.join(', ')}`
+  my = (perms, _ctx) => `I am missing the following permissions: ${perms.map(p => humanReadable[p] ?? p).join(', ')}`,
+  user = (perms, _ctx) => `You are missing the following permissions: ${perms.map(p => humanReadable[p] ?? p).join(', ')}`,
```

Comment:
perms and _ctx are the wrong way around, also change `_ctx` to ctx for both the example and code please.